controllers/src/main/scala/cromwell/pipeline/controller/utils/PathMatchers.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.pipeline.controller.utils
+
+import akka.http.scaladsl.server
+import akka.http.scaladsl.server.PathMatchers.Segment
+import cromwell.pipeline.datastorage.dto
+
+import java.nio.file.{ Path, Paths }
+
+object PathMatchers {
+  val ProjectId: server.PathMatcher1[dto.ProjectId] = Segment.map(dto.ProjectId(_))
+  val Path: server.PathMatcher1[Path] = Segment.map(Paths.get(_))
```

Comment:
That "server.PathMatcher[]" type seems a little bit confusing to me. Maybe it will be better to import akka.http.scaladsl.server.PathMatcher1 and remove "server" part from val's types.

Reply:
Strongly agree. "server"-part looks like something we can get rid of

Reply:
Ok. I dixed it.